Plugin~/WebRTCPlugin/UnityRenderEvent.cpp

```diff
@@ -64,10 +83,17 @@ static void UNITY_INTERFACE_API OnGraphicsDeviceEvent(UnityGfxDeviceEventType ev
     case kUnityGfxDeviceEventInitialize:
     {
         s_mapEncoder.clear();
+
+        IGraphicsDevice* device = GetGraphicsDevice();
+        if (device != nullptr)
+            device->InitV();
```

Comment:
`device->InitV()` is calld in `GetGraphicsDevice()`. 
https://github.com/Unity-Technologies/com.unity.webrtc/blob/feat/vulkan/Plugin~/WebRTCPlugin/GraphicsDevice/GraphicsDevice.cpp#L130
so,It is called twice here,. is this no problem ?